ethpm_types/abi.py

```diff
@@ -84,6 +84,24 @@ def signature(self) -> str:
         return "fallback()"
 
 
+class ReceiveABI(BaseModel):
+    type: Literal["receive"]
+
+    # No `name` field
+    stateMutability: str = "payable"  # NOTE: Should be either "payable" or "nonpayable"
+
+    @property
+    def is_payable(self) -> bool:
+        return self.stateMutability == "payable"
```

Comment:
Should we just always return `True` here explicitly?
And for `fallback` should we return `False` explicitly?

Reply:
Yes, `stateMutability: Literal["payable"]` and this function should return a constant true value